website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -60,6 +61,29 @@ For example, given the forward-to URL `https://my-custom-app.com/api/123`, the `
 - `forward-url-full-path`: `https://my-custom-app.com/api/123`
 - `forward-url-origin`: `https://my-custom-app.com`
 
+## Configuring custom claims
+
+This header can be used to make the Merchant Center API Gateway to include extra public claims in the exchange token sent along with forwarded requests.
+
+The value must be a list of the claims separated by an empty space.
+
+The following claims are supported:
+* **permissions**: Optionally includes the list of applied permissions for the logged in user. The permission values are defined in the format `can<permission-name>`, for example `canViewAvengers`.
```

Comment:
```suggestion
Use the `X-Forward-To-Claims` header to make the Merchant Center API Gateway include extra public claims in the exchange token sent along with forwarded requests.

The value must be a list of the claims separated by a space.

The following claims are supported:
* **permissions**: Optionally includes the list of applied permissions for the logged-in user. The permission values are formatted as `can<permission-name>`, for example `canViewAvengers`.
```

Reply:
Updated in this commit: 9d0903a0fd069c9b485bb1bde48621e7d29fddd1